ra-optimisation/rao-commons/src/test/java/com/farao_community/farao/rao_commons/RaoUtilTest.java

```diff
@@ -59,11 +59,9 @@ public void setUp() {
     }
 
     private void addPtdfParameters(List<String> boundaries) {
-        RaoPtdfParameters raoPtdfParameters = new RaoPtdfParameters();
         if (boundaries != null) {
-            raoPtdfParameters.setBoundariesFromCountryCodes(boundaries);
+            raoParameters.setPtdfBoundariesFromCountryCodes(boundaries);
         }
-        raoParameters.addExtension(RaoPtdfParameters.class, raoPtdfParameters);
     }
```

Comment:
Can you just remove the add method when the argument is null ?

Reply:
done